src/main/scala/it/pagopa/interop/purposemanagement/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -238,34 +238,21 @@ final case class PurposeApiServiceImpl(
     val operationLabel = s"Updating Purpose $purposeId"
     logger.info(operationLabel)
 
-    val update                  = PurposeUpdate.fromApi(uuidSupplier)(payload)
-    val result: Future[Purpose] = commander(purposeId).askWithStatus(UpdatePurpose(purposeId, update, _)).map(_.toAPI)
+    val purposeUpdate = PurposeUpdate.fromApi(uuidSupplier)(payload)
 
-    onComplete(result) { updatePurposeResponse[Purpose](operationLabel)(updatePurpose200) }
-  }
-
-  override def updateDraftPurposeVersion(
-    purposeId: String,
-    versionId: String,
-    updateContent: DraftPurposeVersionUpdateContent
-  )(implicit
-    toEntityMarshallerPurposeVersion: ToEntityMarshaller[PurposeVersion],
-    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
-    contexts: Seq[(String, String)]
-  ): Route = {
-    val operationLabel = s"Updating Draft Version $versionId of Purpose $purposeId"
-    logger.info(operationLabel)
+    val result: Future[Purpose] = for {
+      purpose <- commander(purposeId)
```

Comment:
may I ask you to add a comment like
```
// When refactoring the application, the update of the purpose and the update of the version should be done with a single command and a single event
```
so that we remember to change this behaviour